keyhandler/run.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# *
+# * If not stated otherwise in this file or this component's LICENSE file the
+# * following copyright and licenses apply:
+# *
+# * Copyright 2023 RDK Management
+# *
+# * Licensed under the Apache License, Version 2.0 (the "License");
+# * you may not use this file except in compliance with the License.
+# * You may obtain a copy of the License at
+# *
+# * http://www.apache.org/licenses/LICENSE-2.0
+# *
+# * Unless required by applicable law or agreed to in writing, software
+# * distributed under the License is distributed on an "AS IS" BASIS,
+# * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# * See the License for the specific language governing permissions and
+# * limitations under the License.
+# *
+
+set -e # error out if required
+
+export LD_LIBRARY_PATH=/usr/lib:/lib:/home/root:.
+./ut_core_key_input -p input-event-codes-5.15.yaml -c KEY_SPACE -t EV_KEY -v 1
```

Comment:
`./ut_core_key_input -p input-event-codes-5.15.yaml -c KEY_SPACE -t EV_KEY -v 1`

-> update instructions to check for $1 being empty and dump out an error

`./ut_core_key_input -p input-event-codes-5.15.yaml -c $1 -t $2 -v $3**

or

`./ut_core_key_input -p input-event-codes-5.15.yaml $@`